universal-application-tool-0.0.1/test/repository/ProgramRepositoryTest.java

```diff
@@ -109,4 +111,22 @@ public void updateProgramSync() {
     assertThat(updated.getProgramDefinition().localizedName())
         .isEqualTo(ImmutableMap.of(Locale.US, "new name"));
   }
+
+  @Test
+  public void returnsAllAdmins() throws ProgramNotFoundException {
+    Program withAdmins = resourceCreator.insertActiveProgram("with admins");
+    Account admin = new Account();
+    admin.save();
+    assertThat(repo.getProgramAdministrators(withAdmins.id)).isEmpty();
+    admin.addAdministeredProgram(withAdmins.getProgramDefinition());
+    admin.save();
+    assertThat(repo.getProgramAdministrators(withAdmins.id)).isNotEmpty();
+    assertThat(repo.getProgramAdministrators(withAdmins.id)).containsExactly(admin);
```

Comment:
nit (non-blocking): Feel free to just check that the list just contains `admin` (same below)

Reply:
Sure, can do - easy enough!

Reply:
oh wait no i see what you mean, okay, NOW done